stdlib/tkinter/__init__.pyi

```diff
@@ -403,7 +403,28 @@ class Misc:
     # after_idle is essentially partialmethod(after, "idle")
     def after_idle(self, func: Callable[[Unpack[_Ts]], object], *args: Unpack[_Ts]) -> str: ...
     def after_cancel(self, id: str) -> None: ...
+    if sys.version_info >= (3, 13):
+        def after_info(self, id: str | None = None) -> tuple[str, ...]: ...
+
     def bell(self, displayof: Literal[0] | Misc | None = 0) -> None: ...
+    if sys.version_info >= (3, 13):
+        def tk_busy_cget(self, option: str) -> str: ...
+        busy_cget = tk_busy_cget
+        def tk_busy_configure(self, cnf: dict[str, Any] | str | None = None, **kw: Any) -> Any: ...
+        busy_config = tk_busy_configure
+        busy_configure = tk_busy_configure
+        tk_busy_config = tk_busy_configure
+        def tk_busy_current(self, pattern: str | None = None) -> list[Misc]: ...
+        busy_current = tk_busy_current
+        def tk_busy_forget(self) -> None: ...
+        busy_forget = tk_busy_forget
+        def tk_busy_hold(self, **kw: Any) -> None: ...
+        busy = tk_busy_hold
+        busy_hold = tk_busy_hold
+        tk_busy = tk_busy_hold
+        def tk_busy_status(self) -> bool: ...
+        busy_status = tk_busy_status
+
```

Comment:
There is also #12367 with more precise typing for the busy methods, so let's not add them in this PR.

```suggestion
```